tests/OmniSharp.Tests/HighlightFacts.cs

```diff
@@ -0,0 +1,130 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using OmniSharp.Models;
+using Xunit;
+
+namespace OmniSharp.Tests
+{
+    public class HighlightFacts
+    {
+        [Fact]
+        public async Task HighlightSingleLine()
+        {
+            var workspace = TestHelpers.CreateSimpleWorkspace(new Dictionary<string, string>
+            {
+                { "a.cs", "namespace N1\n{\nclass C1 { int n = true; }\n}" }
+            });
+
+            var controller = new OmnisharpController(workspace, null);
+            var regions = await controller.Highlight(new HighlightRequest() { FileName = "a.cs", Lines = new[] { 2 } });
+            
+            ValidateRegions(regions,
+                new Region("keyword", 2, 0, 5),
```

Comment:
If you add something simple like that it's impossible to test contextual keywords, string interpolation (which by the way I already test), difference between interface names and class names etc.